TRACK_README.md

```diff
@@ -1,11 +1,58 @@
 # Exercism {{LANGUAGE}} Track
 
+![build status](https://travis-ci.org/exercism/xslug.svg?branch=master)
+
 Exercism exercises in {{LANGUAGE}}.
 
-## TODO
+## Setup
+
+The simplest way to install {{LANGUAGE}} is ...
+
+## Contributing
+
+Thank you so much for contributing! :tada:
+
+Please start by reading the general Exercism [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data).
+
+We welcome pull requests that provide fixes and improvements to existing exercises. If you're unsure, then go ahead and open a GitHub issue, and we'll discuss the change.
+
+Please keep the following in mind:
+
+- Pull requests should be focused on a single exercise, issue, or change.
+
+- We welcome changes to code style, and wording. Please open a separate PR for these changes if possible.
+
+- Please open an issue before creating a PR that makes significant (breaking) changes to an existing exercise or makes changes across many exercises. It is best to discuss these changes before doing the work.
```

Comment:
This is great. We may want to point to the [exercism/discussions](https://github.com/exercism/discussions/issues) repo for broader discussions about an exercise.